en/concept/core_components.md

```diff
@@ -0,0 +1,91 @@
+# PX4 Core Components
+
+These "core components" define the core functionality of PX4-Autopilot.
+
+They are tested in continuous integration tests and are regularly monitored and tested by their maintainers.
+Core components are the first priority in stable point releases.
+
+## Core component criteria
+
+Each core component must have:
+
+- Maintainer
+- Tester
```

Comment:
That is desirable, but we haven't had a maintainer for MacOS forever, and many of these others too. So perhaps something less "must" ish

Reply:
Also, who is this tester?